py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -144,7 +144,10 @@ def lambda_handler(event: LambdaDict, context: LambdaContext) -> None:
     logging.info("Context:\n%s", context)
 
     try:
-        batch_args = BatchArgs(**event)
+        if "filesize_threshold" in event:
+            batch_args = BatchArgs(**event)
+        else:
+            batch_args = BatchArgs()
```

Comment:
i think this can just stay as it was now that each member of `BatchArgs` has a default value.